godtools/App/Features/Menu/Domain/UseCases/GetUserActivityStatsUseCase/GetUserActivityStatsUseCase.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GetUserActivityStatsUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 3/20/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
+import SwiftUI
+
+class GetUserActivityStatsUseCase {
+    
+    private let localizationServices: LocalizationServices
+    
+    init(localizationServices: LocalizationServices) {
+        
+        self.localizationServices = localizationServices
+    }
+    
+    func getUserActivityStats(from userActivity: UserActivity) -> [UserActivityStatDomainModel] {
+        
+        let toolOpensStat = UserActivityStatDomainModel(
+            iconImageName: "tool-opens",
+            text: localizationServices.stringForMainBundle(key: "account.activity.toolOpens"),
```

Comment:
For the localizable string keys I remember you had mentioned creating a file for storing those.  Can you go ahead and create a file for storing localizable keys?  
I had created a ```Strings.swift```, but I don't think it's very clean to store all strings in there.  That file could get massive.   It would be more clean to create specific files for storing relevant strings.